Moonscraper Chart Editor/Assets/Scripts/Game/NoteVisuals/NoteVisualsManager.cs

```diff
@@ -36,7 +36,27 @@ public static Note.NoteType GetVisualNoteType(Note note)
         {
             if (Globals.gameSettings.drumsModeOptions == GameSettings.DrumModeOptions.Standard)
             {
-                noteType = Note.NoteType.Strum;
+                if (Globals.gameSettings.drumsLaneCount == 5)
```

Comment:
Oh yeah, forgot to mention I made 5-lane preview use the cymbal textures for lanes Y and O